ccd-definition/CaseField/CaseField-SettleDiscontinue-nonprod.json

```diff
@@ -141,5 +139,38 @@
     "FieldType": "DynamicRadioList",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "partiesNames4",
+    "Label": "**${caseNameHmctsInternal}**",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "isPermissionGranted",
+    "Label": "Has permission been granted by a Judge to discontinue",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "SettleDiscontinueYesOrNoList",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "permissionGrantedComplex",
+    "Label": " ",
+    "FieldType": "PermissionGrantedFields",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "permissionNotGrantedText",
+    "Label": "### Unable to discontinue this claim \n To discontinue from this claim you need to get permission from the court.\n Click cancel to return to the case summary screen and select 'Make\n an application' from the next steps menu and select 'Other'",
```

Comment:
"To discontinue from this claim " should be "To discontinue this claim"

Reply:
Done